jnigen/test/package_resolver_test.dart

```diff
@@ -17,21 +18,28 @@ class ResolverTest {
 void main() async {
   await checkLocallyBuiltDependencies();
   final resolver = Resolver(
-      importMap: {
-        'org.apache.pdfbox': 'package:pdfbox/pdfbox.dart',
```

Comment:
Since this is a breaking change, please bump the version. (It feels a bit ugly to keep supporting the import map, so probably best to do a breaking change.)

I see we haven't released a 0.5 yet, but there are fixes between 0.4 and 0.5, please make a PR to release 0.5 with the fixes in the changelog and then bump to 0.6.0-dev.0 in this PR.